src/repositories/adapters/adapters.ts

```diff
@@ -0,0 +1,12 @@
+import { uuid } from "uuidv4";
+import { EServiceSeed } from "../../model/domain/models.js";
+import { CreateEvent } from "../events.js";
+
+export const eserviceSeedToCreateEvent = (
+  eserviceSeed: EServiceSeed
+): CreateEvent<EServiceSeed> => ({
+  streamId: uuid(),
+  version: 0,
+  type: "EServiceCreated",
```

Comment:
Questo non si chiama così nell'originale mi sembra. Per questa fase ovviamente non ha tanta importanza, ma in generale pensiamo di usare gli stessi eventi che hanno ora giusto? (Direi di sì, altrimenti si complica la migrazione dei dati)